src/reducers.js

```diff
@@ -162,6 +163,13 @@ export const setWasWatered = plotContent =>
 export const resetWasWatered = plotContent =>
   setWasWateredProperty(plotContent, false)
 
+/**
+ * @param {?farmhand.plotContent} plotContent
+ * @returns {?farmhand.plotContent}
+ */
+export const resetWasMined = plotContent =>
+  plotContent && plotContent.itemId === MINED_ITEM_ID ? null : plotContent
```

Comment:
Rather than using items to indicate the shoveled state, I think we want to extend this typedef: https://github.com/jeremyckahn/farmhand/blob/44bde3011e4e33be6c157e2fdae0a6f8c2696a53/src/index.js#L37-L44

I think it would make sense to add a new `farmhand.shoveledPlot` typedef that extends `farmhand.plotContent` similarly to how `farmhand.plotContent` below it does. I think we'd also have to make `farmhand.plotContent`'s `itemId` optional, because there will not be an item associated with a shoveled plot. It might be worth renaming `farmhand.plotContent` to something like `farmhand.plotState`, but don't feel that you need to do that for this PR if it's too much for now (we can smooth out those semantics in a later PR).

Reply:
🔑 thanks for this info! this makes sense to me

Reply:
made these updates